bedrock/newsletter/views.py

```diff
@@ -113,8 +115,18 @@ def confirm_thanks(request):
     return l10n_utils.render(request, "newsletter/confirm.html", {"token_error": token_error, "generic_error": generic_error}, ftl_files=FTL_FILES)
 
 
+def newsletter_all_json(request):
+    """Returns a JSON string of all newsletters configured in Basket."""
+    data = {"newsletters": get_newsletters()}
+    return HttpResponse(json.dumps(data), content_type="application/json")
+
+
 def newsletter_strings_json(request):
-    return l10n_utils.render(request, "newsletter/includes/newsletter-strings.json", content_type="application/json", ftl_files=FTL_FILES)
+    """Returns a JSON string of newsletter IDs mapped to localized titles and descriptions."""
+    newsletters = json.dumps(get_newsletters())
+    return l10n_utils.render(
+        request, "newsletter/includes/newsletter-strings.json", {"newsletters": newsletters}, content_type="application/json", ftl_files=FTL_FILES
```

Comment:
Looking at this template, I don't see it using the `newsletters` that you computed above at all? Is it used in some non-obvious way?

Reply:
Ah yes, good catch! I left this in unintentionally when I was testing things out. Will remove 👍